Lib/test/test_zlib.py

```diff
@@ -511,7 +511,7 @@ def test_flushes(self):
                     b = obj.flush( sync )
                     c = obj.compress( data[3000:] )
                     d = obj.flush()
-                except:
+                except zlib.error:
```

Comment:
> zlib only has single exception(zlib.error)

I doubt it's true, just do ``git grep PyErr_ Modules/zlibmodule.c``.

But point of this except clause is to re-raise an exception, providing info about sync/level parameters.  I would suggest instead rewrite this test with [subTest](https://docs.python.org/3/library/unittest.html#unittest.TestCase.subTest), dropping try-except statement.